packages/graphql/src/translate/queryAST/ast/operations/ConnectionReadOperation.ts

```diff
@@ -163,7 +164,18 @@ export class ConnectionReadOperation extends Operation {
 
         const filtersSubqueries = [...authFilterSubqueries, ...normalFilterSubqueries];
 
-        const aggregationSubqueries = this.aggregationField?.getSubqueries(context) ?? [];
+        // Only add the import if it is nested
+        const isTopLevel = !this.relationship;
```

Comment:
Nitpick: considering the use case of this variable instead assign a variable from a negation and then negate the same variable again I will suggest to convert the variable name to `isNested` and remove all the negations.